### ISSUE TITLE
WinUtil disables useless scheduled tasks?

### ISSUE THREAD

**Issue by rottakore** [ChrisTitusTech/winutil#2824]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
I just want to know if WinUtil disables scheduled tasks, if Yes where do I start this script, because I didn't find anything related to Task Scheduler in the Tweaks tab, if No I would like to know why not, because I would like to have a trustworthy frequently updated task scheduler tweaker.

**Describe alternatives you've considered**
Services to Manual changes Task Scheduler tasks too?

**Additional context**
No aditional context.

**Comment by zmileski** [comment]
https://christitustech.github.io/winutil/dev/tweaks/Essential-Tweaks/Tele/?h=schedule#scheduled-task-changes

**Comment by Marterich** [comment]
TLDR: The "Disable Telemetry" Tweak disables useless Tasks
The rest is answered in the documentation 
/close
